operator/hack/olm-operator-install.sh

```diff
@@ -15,35 +15,28 @@ function main() {
     ;;
   esac
 
-  local -r operator_ns="${1:-}"
-  local -r image_tag_base="${2:-}"
 
   case $# in
-  3)
-    local -r index_version="${3:-}"
-    local -r operator_version="${3:-}"
-    ;;
-  4)
-    local -r index_version="${3:-}"
-    local -r operator_version="${4:-}"
+  5)
+    local -r operator_ns="${1}"
+    local -r index_image_repo="${2}"
+    local -r index_image_tag="${3}"
+    local -r starting_csv_version="${4}"
+    local -r operator_channel="${5}"
     ;;
   *)
-    echo -e "Usage:\n\t$0 [--allow-dirty-tag | -d] <operator_ns> <image_tag_base> <index-version> [<install-version>]" >&2
-    echo -e "Example:\n\t$0 -d index-test quay.io/rhacs-eng/stackrox-operator 3.70.1 3.70.1" >&2
+    echo -e "Usage:\n\t$0 [--allow-dirty-tag | -d] <operator_ns> <index-image-repo> <index-image-tag> <starting-csv-version> <install-channel>" >&2
+    echo -e "Example:\n\t$0 -d index-test quay.io/rhacs-eng/stackrox-operator-index v3.70.1 v3.70.0 latest" >&2
     echo -e "Note that KUTTL environment variable must be defined and point to a kuttl executable." >&2
     exit 1
     ;;
   esac
 
-  check_version_tag "${operator_version}" "${allow_dirty_tag}"
+  check_version_tag "${starting_csv_version}" "${allow_dirty_tag}"
   create_namespace "${operator_ns}"
-  apply_operator_manifests "${operator_ns}" "${image_tag_base}" "${index_version}" "${operator_version}"
-
-  if ! [[ "${USE_MIDSTREAM_IMAGES}" == "true" ]]; then
-    approve_install_plan "${operator_ns}" "${operator_version}"
-  fi
-
-  nurse_deployment_until_available "${operator_ns}" "${operator_version}"
+  apply_operator_manifests "${operator_ns}" "${index_image_repo}" "${index_image_tag}" "${starting_csv_version}" "${operator_channel}"
```

Comment:
Looking at this `apply_operator_manifests` call again, I would suggest you to consider using bash named arguments. Optional for this PR.
See https://redhat-internal.slack.com/archives/CELUQKESC/p1692118560975079

Reply:
Given this function has just one caller, and all args are required, I'm not seeing the benefit, TBH.
Especially given that in the caller there would be a lot of repetition:
`apply_operator_manifests "operator_ns=${operator_ns}" "index_image_repo=${index_image_repo}" "index_image_tag=${index_image_tag}" ...`